content/pt/docs/concepts/semantic-conventions.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Convenção Semântica
+description: Nomes comuns para diferentes tipos de operações e dados.
+weight: 30
+default_lang_commit: 9c679ccf62092a34847a01cf97ba0f92847021c0
+---
+
+O OpenTelemetry define [Convenção Semântica](/docs/specs/semconv/), algumas vezes
+chamado de atributos semânticos, que especificam nomes comuns para diferentes tipos de
+operações e dados. O benefício de usar convenções semânticas é seguir um
+padrão de nomenclatura comum que pode ser padronizado em uma base de código, bibliotecas e
+plataformas.
+
+Convenções Semânticas estão disponíveis para rastros, métricas, logs e recursos:
```

Comment:
Hum, "recursos" aqui soa meio estranho, nao? Ainda mais sendo que usamos "recursos" para outras coisas, completamente diferentes 

![image](https://github.com/user-attachments/assets/a021673b-6762-4512-9de2-a100f32a546a)

Reply:
Podemos pensar em uma alternativa, pois está referenciando atributos de recursos.

Reply:
Eu imagino que traducao aqui mais atrapalha do que ajuda, nao sei se tem alguma palavra que represente o que o "resource" representa em OTel. 🤔

Reply:
Eu deixaria sem tradução. Se for traduzir, sugiro usar "atributos de recursos". Na própria documentação temos várias referencias sobre isso. Uma delas é o uso da variável de ambiente `OTEL_RESOURCE_ATTRIBUTES`. Imagina se essa variável fosse `OTEL_RESOURCE`. Recurso de que? Ficaria meio vago.

Reply:
Creio que podemos deixar sem tradução, semelhante ao "logs", pra não confundir com as outras menções de recursos.